tests/src/test/scala/org/http4s/multipart/MultipartParserSpec.scala

```diff
@@ -97,13 +130,11 @@ object MultipartParserSpec extends Specification {
               |catch me if you can!
               |""".stripMargin)
 
-      def unspool(str: String): Process0[ByteVector] = emit(ByteVector view (str getBytes "ASCII"))
+      val preamble = emit(ByteVector.view("Misery is the river of the world".getBytes("ASCII"))).repeat.take(10)
+      val epilogue = emit(ByteVector.view("Everybody Row!\n".getBytes("ASCII"))).repeat.take(10)
 
-      val results: Process0[Headers \/ ByteVector] = unspool(input) pipe MultipartParser.parse(boundary)
-
-      val bytes = results.toVector collect {
-        case \/-(bv) => bv
-      }
+      val results: Process0[Headers \/ ByteVector] =
+        (preamble ++ emit(ByteVector.view("\r\n".getBytes("ASCII"))) ++ unspool(input, 15) ++ epilogue).pipe(MultipartParser.parse(boundary))
```

Comment:
This test shows the requirement listed in my long rant above of the requirement for the single CRLF after the Preamble.